### ISSUE TITLE
Breaking change in Chromium 94+

### ISSUE THREAD

**Issue by jacksteamdev** [extend-chrome/rollup-plugin-chrome-extension#115]
# tldr; Breaking change in Chromium 94

Please upgrade rollup-plugin-chrome-extension to a version >= 3.6.0:

```sh
npm i rollup-plugin-chrome-extension@3.6 -D
```

## Background

Chromium 94 introduced a breaking change that affects the dynamic import this plugin (rollup-plugin-chrome-extension or RPCE) uses to load content scripts.  Dynamic imports in content scripts were relative to the Chrome Extension origin in Chromium <= 93. In Chromium 94, dynamic imports are relative to the host page origin, so content scripts must use `chrome.runtime.getURL` to construct a url that uses the `chrome-extension://` scheme.

This issue was fixed in the latest version of RPCE, but extensions built with previous versions will encounter the following error for content scripts:

![image](https://user-images.githubusercontent.com/23390212/132536405-ebe03bc8-416e-4309-9d10-19f63da832c9.png)

Future versions of RPCE (>= 4.0.0) will not use dynamic imports by default.
